package.json

```diff
@@ -37,7 +37,7 @@
     "tslint": "^5.12.0",
     "typedoc": "^0.15.0-0",
     "typedoc-plugin-external-module-map": "^1.0.0",
-    "typescript": "~3.5.3",
+    "typescript": "^3.9.1-rc",
```

Comment:
Do not use `^`, we should either pin or use `~`. Right now it is not as bad with the `-rc`, but it would be better still.

Reply:
Why? it is default for npm/yarn. Is there anything particular what breaks? If not we can pin when it breaks.

Reply:
I'm supposed to run `yarn upgrade` from time to time to align deps of the main repo with what would be used by new downstream apps, and it broke the last time when we had `^3.5.x` because it pulled `3.7.x` which was having issues on our repo.

So yes, it's not really important for consumers, but the range itself makes more sense and it helps a little bit when upgrading.

Reply:
I think we should use latest typescript and `yarn upgrade` should bring it. If there are some issues during upgrade then we should temporary pin.